backend-go/samples/sub/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"github.com/nats-io/nats.go"
+	"github.com/wuespace/telestion/backend"
+	"log"
+	"os"
+	"os/signal"
+)
+
+const (
+	Subject = "publish_fast"
+)
```

Comment:
Let's maybe rename this option to `MonitoredSubject` or something like that.

As it is, this is the first thing the eye jumps to in this sample and it's weird to read `Subject = "publish_fast"` in a sample that's all about subscribing.

Alternatively, it might be a good idea to just rename this subject to `"sample_data"` or something similar everywhere to take away any remaining confusion.

Reply:
Fixed in c4cc8e05801472bf157544248dbfd6d77397bc9b